modules/cnv-deploying-cnv.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_install/installing-container-native-virtualization.adoc
+
+[id="cnv-deploying-cnv_{context}"]
+= Deploying {ProductName}
+
+After subscribing to the *KubeVirt HyperConverged Cluster Operator* catalog,
+create the *KubeVirt HyperConverged Cluster Operator Deployment* custom resource
+to deploy {ProductName}.
+
+.Prerequisites
+
+* You must be subscribed to the *KubeVirt HyperConverged Cluster Operator* catalog
+in the `kubevirt-hyperconverged` namespace.
+
+.Procedure
+
+. Navigate to the *Catalog* -> *Installed Operators* page.
+
+. Under *Provided APIs*, locate the *KubeVirt HyperConverged Cluster Operator Deployment*
```

Comment:
Is it 'under' or 'in'?

Reply:
It's physically under the page title "Provided APIs" so I *think* it should be "under." You don't click a button or anything to get there once you're on the Installed Operators page.

Reply:
Please check the IBM Style Guide's section about "Graphical user interface elements." It sounds like it might be what ISG considers a "pane," which would mean "in" wins.